.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Which commands were invoked
+2. Where the error appears
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. Debian Bullseye]
+ - Version [e.g. 0.4]
+ - Bash Version [e.g. 5.1.4]
+
```

Comment:
Should we add a kw version?
I know we don't have a release yet but this should be useful in the future after we release kw 1.0